### PR TITLE
Cypress/E2E: Improve E2E report

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Improve E2E reporting
- correct showing of Docker image and TM4J links
- introduce TM4J environment variables for flexibility when saving records to TM4J

### CODE REVIEW

File: e2e/save_report.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ const saveReport = async () => {
         DIAGNOSTIC_TEAM_ID,
         FAILURE_MESSAGE,
         TM4J_ENABLE,
+        TM4J_CYCLE_KEY,
```
Comment (furqanmlk) [inline e2e/save_report.js:59]: Thanks @saturninoabril 
We are till using ```TM4J``` name ? I thought it has changed !

**Review by furqanmlk — APPROVED** [review]

File: e2e/save_report.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/save_report.js:59]: 0/5 on name to use. Zephyr sounds unfamiliar to me :)

**Review by prapti — APPROVED** [review]
Looking great!

**Pull request merged**